easel/lib/widgets/easel_price_input_field.dart

```diff
@@ -34,42 +29,38 @@ class EaselPriceInputField extends StatelessWidget {
         ),
         SizedBox(height: 4.h),
         Stack(
+          alignment: Alignment.centerLeft,
           children: [
             Positioned(
-              child: Image.asset(kTextFieldSingleLine,
-                  width: 1.sw,
-                  height: isTablet ? 32.h : 40.h,
-                  fit: BoxFit.fill),
+              child: Image.asset(kTextFieldSingleLine, width: 1.sw, height: isTablet ? 32.h : 40.h, fit: BoxFit.fill),
             ),
-            Row(
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: [
-                Expanded(
-                    child: TextFormField(
-                        style: TextStyle(
-                            fontSize: 18.sp,
-                            fontWeight: FontWeight.w400,
-                            color: EaselAppTheme.kDarkText),
-                        controller: controller,
-                        validator: validator,
-                        minLines: 1,
-                        maxLines: 1,
-                        keyboardType: TextInputType.number,
-                        textCapitalization: TextCapitalization.none,
-                        inputFormatters: inputFormatters,
-                        decoration: InputDecoration(
-                            hintText: "price_hint".tr(),
-                            hintStyle: TextStyle(
-                                fontSize: 15.sp,
-                                fontWeight: FontWeight.w400,
-                                color: EaselAppTheme.kGrey),
-                            border: const OutlineInputBorder(
-                                borderSide: BorderSide.none),
-                            floatingLabelBehavior: FloatingLabelBehavior.always,
-                            contentPadding:
-                                EdgeInsets.fromLTRB(10.w, 0.h, 10.w, 0.h)))),
-                const _CurrencyDropDown()
-              ],
+            Align(
```

Comment:
Why you aligned this . What it will solve?

Reply:
Updated.

Reply:
Yes you updated but what it fixes in the first place?

Reply:
I fixed the currency button in the price screen as you mentioned, but then we realized fixed that issue too so we went back to Git history and revert that code & from where that Align came but when I removed it the button didn't not show any change.